packages/frontend/src/components/BottomBar.tsx

```diff
@@ -0,0 +1,49 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faHouse,
+  faMagnifyingGlass,
+  faPerson,
+  faTrophy,
+} from "@fortawesome/free-solid-svg-icons";
+export function BottomBar(_: {}) {
+  return (
+    <div className="flex justify-between grid-cols-4 mb-3 bottom-0 left-0 right-0 fixed w-full bg-squash-yellow">
```

Comment:
Comparing with Figma, you are missing the top border here (and padding too!).

Also, there are some unnecessary classes here:
- `right-0` is not essential since your element is `100%` wide, and also `left: 0`.
- I'm not sure what `bg-squash-yellow` is trying to achieve here...

Here's how you'd add it:
```suggestion
    <div className="flex justify-between grid-cols-4 mb-3 pt-3 bottom-0 left-0 fixed w-full border-t-p-grey-900 border-t-4">
```

* `border-4` and `border-t-p-grey-900` will make the border match Figma.
* The `pt-3` pads the top of the bar, so there's space between the border and the buttons.